README.md

```diff
@@ -60,18 +60,21 @@ out when using this package.
     ```go
     import (
         _ "crypto/sha256"
-   	    _ "crypto/sha512"
+        _ "crypto/sha512"
     )
     ```
     This may seem inconvenient but it allows you replace the hash 
     implementations with others, such as https://github.com/stevvooe/resumable.
  
-2. Even though `digest.Digest` may be assemable as a string, _always_ 
+2. Even though `digest.Digest` may be assemblable as a string, _always_ 
     verify your input with `digest.Parse` or use `Digest.Validate`
     when accepting untrusted input. While there are measures to 
     avoid common problems, this will ensure you have valid digests
     in the rest of your application.
 
+3. While alternative encodings of hash values (digests) are possible (for
+    example, base64), this package deals exclusively with hex-encoded digests.
```

Comment:
Let's remove this section. The doc update clarifies the point and the interface has been updated to be less opinionated about encoding.  Other sections look good.